### PR TITLE
test: improve common package coverage

### PR DESCRIPTION
**Pull request by skosito**
# Description

note: too many files changed in diff because i moved some files (81 receipts.json files) and added files for transactions data (81 tx.json files)

- increases codecov for common package from 36.1% to 91.6% (some parts are tricky to test, but should be good enough coverage)
- adds tx files for ethereum proof tests, same block as receipts
- moves some data to testdata to avoid duplication, but still there is todo with existing issue to centralize it further
- remove some unused code
- simplifies some small parts

Closes: #1833 

## Type of change

New tests

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Include instructions and any relevant details so others can reproduce. 

- [ ] Tested CCTX in localnet
- [ ] Tested in development environment
- [x] Go unit tests
- [ ] Go integration tests
- [x] Tested via GitHub Actions 

# Checklist:

- [x] I have added unit tests that prove my fix feature works

### CODE REVIEW

**Review by lumtis — APPROVED** [review]
This looks good to me, great 👍
I think we can also do https://github.com/zeta-chain/node/issues/1905 for the code structure. But this can be done in another PR

File: common/tss_test.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+	pk, err := cosmos.Bech32ifyPubKey(cosmos.Bech32PubKeyTypeAccPub, pubKey)
+	require.NoError(t, err)
+	testCases := []struct {
+		name          string
+		tssPubkey     string
+		bitcoinParams *chaincfg.Params
+		wantErr       bool
+	}{
+		{
+			name:          "Valid TSS pubkey",
```
Comment (kingpinXD) [inline common/tss_test.go:57]: Maybe we should add a test case for mainnet params ,

File: common/tss_test.go [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+)
+
+func TestGetTssAddrEVM(t *testing.T) {
+	_, pubKey, _ := testdata.KeyTestPubAddr()
+	pk, err := cosmos.Bech32ifyPubKey(cosmos.Bech32PubKeyTypeAccPub, pubKey)
+	require.NoError(t, err)
+	testCases := []struct {
+		name      string
+		tssPubkey string
+		wantErr   bool
```
Comment (kingpinXD) [inline common/tss_test.go:19]: Maybe we can add an expected address to and assert that as well?

**Review by kingpinXD — APPROVED** [review]
left minor comments 
LGTM

**Pull request merged**